File: backend/src/main/java/com/romeo/VetLink/subscription/service/dtos/SubscriptionDTO.java
package com.romeo.VetLink.subscription.service.dtos;

import com.romeo.VetLink.shopItem.service.dtos.ShopItemDTO;
import lombok.Data;

import java.util.List;

@Data
public class SubscriptionDTO {
    Integer id;
    private String name;
    private String recurrence;
    private String price;
    private String shortDescription;
    private List<ShopItemDTO> shopItemDTOs;
}

File: frontend/src/app/subscription-section/domain/subscription-dto.ts
import {ShopItemDto} from "../../shop-section/domain/shop-item-dto";

export class SubscriptionDto{
  id : number | null = null;
  name: string | null = null;
  price: string | null = null;
  recurrence: string | null = null;
  shortDescription: string | null = null;
  shopItemDTOs: ShopItemDto[] = [];
  owner: string | null = null;

}
